adoc/advanced_topics_proxy_quickstart.adoc

```diff
@@ -528,7 +529,7 @@ This upgrade is similar to a standard client migration.
 .Synchronizing Target Channels
 ====
 Before successfully initializing the product migration, you first must make sure that the migration target channels are completely mirrored.
-For the upgrade to {productname} 3.2 Proxy, at least the [systemitem]``SUSE Linux Enterprise Server 12 SP3`` base channel with the [systemitem]``SUSE Manager Proxy 3.2`` child channel for your architecture is required.
+For the upgrade to {productname} 3.2 Proxy, at least the [systemitem]``SUSE Linux Enterprise Server 12 SP4`` base channel with the [systemitem]``SUSE Manager Proxy 3.2`` child channel for your architecture is required.
```

Comment:
reword: "To upgrade to {productname} 3.2 Proxy, you will require at least the [systemitem]``SUSE Linux Enterprise Server 12 SP4`` base channel with the [systemitem]``SUSE Manager Proxy 3.2`` child channel for your architecture."